### PR TITLE
update release-comms mailing list

### PR DESCRIPTION
**Pull request by mkorbi**
Updating the release-comms mailing list with the new Release Lead + Shadows and Comms Team

### CODE REVIEW

File: groups/groups.yaml [hunk 1]
```diff
@@ -1144,12 +1144,16 @@ groups:
     members:
       - alarcj137@gmail.com
       - chu.karen.h@gmail.com
-      - k8s@thatmightbe.com
+      - k8s@thatmightbe.
```
Comment (justaugustus) [inline groups/groups.yaml:1147]:
```suggestion
      - k8s@thatmightbe.com
```

**Comment by spiffxp** [comment]
/hold cancel
I'm assuming this is good to go, please ping me on slack or assign a followup issue here if not

**Comment by spiffxp** [comment]
Ran `make -C groups run -- --confirm`

**Comment by justaugustus** [comment]
Thanks Aaron!

**Pull request merged**